### ISSUE TITLE
BuildEnvironmentHelper s_visualStudioProcess has broken XML comment

### ISSUE THREAD

**Issue by elachlan** [dotnet/msbuild#5580]
```
/// <summary>
        /// Name of the Visual Studio (and Blend) process.
        // VS ASP intellisense server fails without Microsoft.VisualStudio.Web.Host. Remove when issue fixed: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/574986
        /// </summary>
        private static readonly string[] s_visualStudioProcess = {"DEVENV", "BLEND", "Microsoft.VisualStudio.Web.Host"};
```

**Comment by elachlan** [comment]
I missed the PR which fixes it in #5575

**Comment by BenVillalobos** [comment]
Closing per:

> I missed the PR which fixes it in #5575
